src/agent/onefuzz/src/process.rs

```diff
@@ -7,6 +7,13 @@ use std::path::Path;
 use std::process::Command;
 use std::time::Duration;
 use std::{collections::HashMap, process::Stdio};
+use tokio::{
+    io::{AsyncBufReadExt, AsyncRead, BufReader},
+    process::Child,
+    sync::Notify,
+};
+
+const MAX_LOG_LINE_LENGTH: usize = 8192;
 
```

Comment:
Could you comment on how this value was chosen? Even if arbitrary, helpful to keep track of.